packages/blade/src/utils/makeBezier/makeBezier.native.ts

```diff
@@ -0,0 +1,8 @@
+import { Easing } from 'react-native-reanimated';
+import type { EasingFunctionFactory } from 'react-native-reanimated';
+
+const makeBezier = (x1: number, y1: number, x2: number, y2: number): EasingFunctionFactory => {
```

Comment:
nit: can we make it accept object as argument? i guess we have been following this for function accepting more than one arguments
```suggestion
const makeBezier = ({x1, y1, x2, y2}:{x1: number, y1: number, x2: number, y2: number}): EasingFunctionFactory => {
```

Reply:
no generic input type for `makeBezier` function? I see you did it for web

```
const makeBezier = <X1 extends number, Y1 extends number, X2 extends number, Y2 extends number>(
  x1: X1,
  y1: Y1,
  x2: X2,
  y2: Y2,
)
```

Reply:
We can do this but I don't see the value in doing it for native. On web we're defining these generics so that we can form a return type of `cubic-bezier(${X1}, ${Y1}, ${X2}, ${Y2})` out of it. On native the return type is always `EasingFunctionFactory`. Am I missing something here?

Reply:
then to I would suggest why do we even need generics for web? we can just do

```
const makeBezier = (
  x1: number,
  y1: number,
  x2: number,
  y2: number,
): `cubic-bezier(${number}, ${number}, ${number}, ${number})` => {
  return `cubic-bezier(${x1}, ${y1}, ${x2}, ${y2})`;
};
```

Reply:
Generics makes sense for web since I can enforce that my return value consists the input values in the exact expected sequence of `x1, y1, x2, y2`